web/src/components/layout/Container.vue

```diff
@@ -1,11 +1,12 @@
 <template>
-  <div class="w-full p-4 mx-auto" :class="{ 'max-w-5xl': !fullWidth, 'md:px-0': fullWidth }">
+  <div class="w-full p-4 mx-auto" :class="{ 'max-w-5xl': !fullWidth && !fillWidth, 'md:px-0': fullWidth }">
```

Comment:
Do we really want to use the full page width here? We don't need it and all other tabs like config, warnings, etc also dont have it.

Reply:
I have changed it only here as otherwise it will look
![attels](https://github.com/woodpecker-ci/woodpecker/assets/165205/cf3b08c4-2976-4d4a-b6c2-f950a0c9e538)

vs 

![attels](https://github.com/woodpecker-ci/woodpecker/assets/165205/f484c81e-10ee-4cfb-a096-4abe24d8e32e)

so when you click between steps it's jumping around that does not look consistent and jumpy

Reply:
after this change log view and error message has same size and when you click between steps it is same size at least in width